<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entity Line Tester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Entity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="simQt::EntityLineEdit" name="entityLine"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="platformButton">
       <property name="text">
        <string>Add Platform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="beamButton">
       <property name="text">
        <string>Add Beam</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gateButton">
       <property name="text">
        <string>Add Gate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addManyButton">
       <property name="text">
        <string>Add 10000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renameButton">
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>simQt::EntityLineEdit</class>
   <extends>QWidget</extends>
   <header>simQt/EntityLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
